api/envoy/extensions/load_balancing_policies/client_side_weighted_round_robin/v3/client_side_weighted_round_robin.proto

```diff
@@ -68,4 +69,9 @@ message ClientSideWeightedRoundRobin {
   // calculated as eps/qps. Configuration is rejected if this value is negative.
   // Default is 1.0.
   google.protobuf.FloatValue error_utilization_penalty = 6 [(validate.rules).float = {gte: 0.0}];
+
+  // By default, endpoint weight is computed based on the `application_utilization <https://github.com/cncf/xds/blob/88aa3b3c978aa0d37c1df3b075f791855df80203/xds/data/orca/v3/orca_load_report.proto#L57>`_ field reported by the endpoint.  If that field is not set, then utilization
+  // will instead be computed by taking the max of the values of the metric names (e.g. "foo", "bar") specified here.  If none of the specified metrics are present in the `named_metrics <https://github.com/cncf/xds/blob/88aa3b3c978aa0d37c1df3b075f791855df80203/xds/data/orca/v3/orca_load_report.proto#L49>`_ map, then
```

Comment:
I don't think this field should be specific to the ORCA `named_metrics` field; I think it should be capable of grabbing any field out of the ORCA proto, just like the [`lrs_report_endpoint_metrics` field](https://github.com/envoyproxy/envoy/blob/1eedeeed1a421f15577733347e8a32c1ace1f953/api/envoy/config/cluster/v3/cluster.proto#L1181) that we added to CDS.  I think we want this to have the same semantics as that field, both for consistency and because it may be useful to include top-level ORCA fields like `cpu_utilization` and `mem_utilization`.

Reply:
Done. I've updated the comment.